models/Element/AbstractElement.php

```diff
@@ -238,9 +240,6 @@ public function getParent(): ?AbstractElement
         return $this->parent;
     }
 
-    /**
```

Comment:
Why removed this phpDoc ?

Reply:
Is redundant because its already set in ElementInterface

Reply:
> Is redundant because its already set in ElementInterface

👍